src/resources/packages/classy/style.pcss

```diff
@@ -555,13 +555,14 @@
 		}
 
 		.classy-modal__content {
+			--classy-input-width: 444px;
```

Comment:
Let's move this to the root element.

Reply:
This particular one is already in the root element, but here it is an override of the root element.

Reply:
Thanks for clarifying. It looks a bit suspicious to have this override here, but it's fine for the time being.